recipes/goatools/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "goatools" %}
+{% set version = "1.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/goatools-{{ version }}.tar.gz
+  sha256: 1b5635481221e42bf2d8efc4ebdca42cd7696b52f52ce5d32fe3eb29d447cdb2
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

Comment:
For future reference this package has scripts and cannot be noarch.

Reply:
Thanks @ocefpaf, for merging this. Can you explain why a package with scripts can't be `noarch`? I am not sure I understand the reason.

Reply:
Yep. Scripts are a problem on Windows, and they are also a deprecated form of CLI. If you send a PR upstream to change that to entrypoints then we can make it noarch.

Reply:
ok thanks for the explanation.